normandy/control/static/control/js/components/action_forms/HeartbeatForm.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { reduxForm } from 'redux-form'
+import { _ } from 'underscore'
+import { formatLabel } from '../../utils/formHelpers.js'
+
+
+class SurveyForm extends React.Component {
+  render() {
+    const { selectedSurvey, fields, showDefaults } = this.props;
+    const surveyObject = selectedSurvey || fields.defaults;
+    let headerText = 'Default Survey Values';
+    let containerClass = 'fluid-8';
+    if (selectedSurvey) {
+      headerText = selectedSurvey.title.initialValue || 'New survey';
+      containerClass += ' active';
+    }
+
+    return (
+      <div id='survey-form' className={containerClass}>
+        { selectedSurvey &&
+          <span className="return-to-defaults" onClick={showDefaults}>
+            <i className="fa fa-long-arrow-left pre"></i> Return to defaults
+          </span>
+        }
+        <h4>{headerText}</h4>
+        {
+          Object.keys(surveyObject).map(fieldName =>
```

Comment:
It's a little bit easier to iterate through the list with `Object.keys` right now because we toggle between showing the default fields and a selected survey's fields. Selected surveys will have a title & weight field associated with them, defaults will not. So we'd need to add some conditional logic in to determine whether or not those should be rendered if we decide to hardcode the fields rather than iterating the keys.